src/app.postcss

```diff
@@ -11,6 +12,10 @@
 		scroll-behavior: auto;
 	}
 }
+@font-face {
+	font-family: "Open Sans";
+	src: url("/fonts/OpenSans-VariableFont_wdth,wght.ttf") format('ttf');
```

Comment:
ttf files can be converted/compressed to woff & woff2 formats

which are created specifically for web fonts